neo/Consensus/RegenerationMessage.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.IO;
+using Neo.Network.P2P.Payloads;
+using Neo.IO;
+
+namespace Neo.Consensus
+{
+    internal class Renegeration : ConsensusMessage
+    {
+        /// <summary>
+        /// Original PrepareRequest in which, at least, M nodes signed
+        /// </summary>
+        public ConsensusPayload PrepareRequestPayload; 
+        /// <summary>
+        /// Partial signatures of, at least, M nodes
+        /// </summary>
+        public byte[][] SignedPayloads;
+
+        /// <summary>
+        /// Constructors
+        /// </summary>
+        public Renegeration() : base(ConsensusMessageType.Renegeration) { }
+
+        public override void Deserialize(BinaryReader reader)
+        {
+            base.Deserialize(reader);
+
+            PrepareRequestPayload = new ConsensusPayload();
+            ((ISerializable)PrepareRequestPayload).Deserialize(reader);
+
+            int nValidators = reader.ReadInt32();
+
+            for (int sp = 0; sp < nValidators; sp++)
```

Comment:
We need to initial `SignedPayloads` before signing any value to it because it is going to throw exception.
```
SignedPayloads = new byte[nValidators][];
```

Reply:
@wanglongfei88, I would take me some debugging time...
Thanks